README.md

```diff
@@ -2,272 +2,36 @@
 
 ## Configuration
 
-Check out
-[`web/modules/custom/hoeringsportal_deskpro/README.md`](web/modules/custom/hoeringsportal_deskpro/README.md)
-for details on configuring the Deskpro integration.
+### Guide for local development README
 
-### OpenID Connect
+[Local development](web/documentation/localDevelopment.md)
```

Comment:
It is customary to put documentation in a `docs` folder in the root of the project rather than in `web/documentation`.

Reply:
Moved